Plugin/CppApi/include/ArcGISCompassController.h

```diff
@@ -53,7 +49,10 @@ class TOOLKIT_EXPORT ArcGISCompassController : public AbstractTool
   bool setView(Esri::ArcGISRuntime::SceneQuickView* sceneView);
 
   double heading() const;
+  void setHeading(const double value);
+
   bool autoHide() const;
+  void setAutoHide(const bool value);
```

Comment:
Both setters should just take args by copy, not const.

Reply:
That's what I have.  Pushed incorrect commit :(